spec/workers/commit_monitor_handlers/commit_range/rubocop_checker/message_builder_spec.rb

```diff
@@ -22,12 +22,12 @@
 4 files checked, 4 offenses detected
 
 **spec/workers/commit_monitor_handlers/commit_range/rubocop_checker/data/#{rubocop_check_directory}/coding_convention.rb**
-- [ ] :exclamation: - [Line 3](https://github.com/some_user/some_repo/blob/8942a195a0bfa69ceb82c020c60565408cb46d3e/spec/workers/commit_monitor_handlers/commit_range/rubocop_checker/data/#{rubocop_check_directory}/coding_convention.rb#L3), Col 5 - [Layout/AlignHash](http://rubydoc.info/gems/rubocop/#{rubocop_version}/RuboCop/Cop/Layout/AlignHash) - Align the elements of a hash literal if they span more than one line.
-- [ ] :exclamation: - [Line 4](https://github.com/some_user/some_repo/blob/8942a195a0bfa69ceb82c020c60565408cb46d3e/spec/workers/commit_monitor_handlers/commit_range/rubocop_checker/data/#{rubocop_check_directory}/coding_convention.rb#L4), Col 5 - [Layout/AlignHash](http://rubydoc.info/gems/rubocop/#{rubocop_version}/RuboCop/Cop/Layout/AlignHash) - Align the elements of a hash literal if they span more than one line.
+- [ ] :exclamation: - [Line 3](https://github.com/some_user/some_repo/blob/8942a195a0bfa69ceb82c020c60565408cb46d3e/spec/workers/commit_monitor_handlers/commit_range/rubocop_checker/data/#{rubocop_check_directory}/coding_convention.rb#L3), Col 5 - [Layout/HashAlignment](http://rubydoc.info/gems/rubocop/#{rubocop_version}/RuboCop/Cop/Layout/HashAlignment) - Align the keys and values of a hash literal if they span more than one line.
```

Comment:
Note:  This needed to be regenerated because the class was changed from `AlignHash` to `HashAlignment`, and the `COP_URIS` constant:

https://github.com/ManageIQ/miq_bot/blob/995fc9fb25c46066dfe48d09a7e1eb31d1aebeef/app/workers/commit_monitor_handlers/commit_range/rubocop_checker/message_builder.rb#L32-L38

in the `MessageBuilder` as not correct.

Regenerated the example results for the specs as described here:

https://github.com/ManageIQ/miq_bot/blob/995fc9fb25c46066dfe48d09a7e1eb31d1aebeef/spec/support/rubocop_spec_helper.rb#L1-L14

locally with the config file changes for https://github.com/ManageIQ/guides/pull/421 applied.